akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -283,6 +284,39 @@ final case class Intersperse[T](start: Option[T], inject: T, end: Option[T]) ext
 /**
  * INTERNAL API
  */
+private[akka] final case class Dedupe[T]() extends GraphStage[FlowShape[T, T]] {
```

Comment:
Please use either (T,T) => Boolean or a scala.math.Equiv[T] instance here so we don't have to rely on "universal equality"